js/src/personal/riyuan/components/experience/TimelineItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Text, Timeline } from '@mantine/core'
+import { ReactNode } from 'react'
+
+interface TimelineItemProps {
+  bullet: ReactNode
+  title: string
+  description: string
+  time: string
+  lineVariant?: 'dashed' | 'solid'
+}
+
+export function TimelineItem({
+  bullet,
+  title,
+  description,
+  time,
+  lineVariant,
+}: TimelineItemProps) {
+  return (
+    <Timeline.Item
+      bullet={bullet}
+      title={title}
+      lineVariant={lineVariant}
+    >
+      <Text
+        c="dimmed"
+        size="sm"
+        dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
Can this be done without using `dangerouslySetInnerHTML`?

Reply:
Done it without using danerouslysetinnerhtml